src/Java.Interop/Java.Interop-MonoAndroid.csproj

```diff
@@ -50,6 +50,7 @@
   <ItemGroup>
     <Compile Include="**\*.cs" />
     <Compile Remove="Tests\**\*.cs" />
+    <Compile Remove="obj*\**\*.cs" />
```

Comment:
You probably shouldn't remove *everything* under the obj directories. You'll break any kind of auto-generation.

I think what you want is: `<Compile Include="**\*.cs" Exclude="obj*\**\*.cs" />`. That'll include the files you want *without* removing all the files you should not be removing.